### DISCUSSION TITLE
Directions please...

### DISCUSSION THREAD

**Discussion by giggywithit** [alsa-project/alsa-utils#147] [General]
Hello

I design and build audio equipment and would like to know where or how I would go about getting access to the sound states to control a GPIO for a raspberry Pi.

What I would like ideally is to control a GPIO state based upon actual non-zero data playback.
My desire is to use the GPIO to control a mute function to silence the interactive periods in DAC activity.
I know little to nothing about programming but imagine the function is intelligent enough to discern the difference between an open device and actual music playback.
Other methods keep open the device to keep from switching off and on, however, I am looking for a more professional and detailed method to create.

Any pointers helpful
thank you
S

**Comment by perexg** [comment]
It depends on the used applications. The PCM stream may be redirected in the user space (for example: sound daemon or alsa-lib's file (tee) PCM plugin).

**Comment by beedell-roke** [comment]
Instead of accepting questions as issues, why not enable the discussion-tab for this repository?
